cmd/kubeadm/app/phases/addons/dns/dns_test.go

```diff
@@ -1383,6 +1383,75 @@ func TestCreateDNSService(t *testing.T) {
 	}
 }
 
+func TestDeployedDNSAddon(t *testing.T) {
+	newMockClientWithImage := func(t *testing.T, img string) *clientsetfake.Clientset {
+		ctx := context.Background()
+		t.Helper()
```

Comment:
```suggestion
		t.Helper()
		ctx := context.Background()
```
Putting `t.Helper()` front seems clearer